### PR TITLE
Fixes #3707

### LINKED ISSUE #3707: Adding plugin items to toolbar then removing the plugin causes server to crash w/ no explanation
For example see https://github.com/JohnMcLear/ep_align/issues/13
Add these to settings.json
Restart Server
Works
Remove ep_align plugin
Restart Server
Dead server.

### PR DESCRIPTION
**Pull request by JohnMcLear**
https://github.com/ether/etherpad-lite/issues/3707

Also reports the error to the server admin / plugin dev via console.warn.

I can't see any security considerations here but haven't done thorough testing.

### CODE REVIEW

File: src/node/utils/toolbar.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ ButtonsGroup.prototype.render = function () {
   }
 
   return _.map(this.buttons, function (btn) {
-    return btn.render();
+    if(btn) return btn.render();
```
Comment (JohnMcLear) [inline src/node/utils/toolbar.js]: This is probably not needed.

File: src/node/utils/toolbar.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ ButtonsGroup.prototype.addButton = function (button) {
 };
 
 ButtonsGroup.prototype.render = function () {
-  if (this.buttons.length == 1) {
+  if (this.buttons && this.buttons.length == 1) {
```
Comment (JohnMcLear) [inline src/node/utils/toolbar.js]: Again maybe not needed but maybe worth having just for some weirdness

**Comment by JohnMcLear** [comment]
Any reason we can't merge @muxator ?

**Comment by muxator** [comment]
Sure we can merge it.

If possible, next time please do a PR from a personal fork, so it can eventually be reworked without interfering with the main repo. Or directly commit to master if you think there is no need for review. :)

**Pull request merged**